integration-tests/src/test/java/org/web3j/generated/MetaCoin.java

```diff
@@ -1,153 +0,0 @@
-package org.web3j.generated;
```

Comment:
How come this is deleted? I believe it's used for one of the Truffle integration tests.